labelmerge/resources/zona_bb_subcortex/convexHull_roi.py

```diff
@@ -1,11 +1,12 @@
 #!/bin/env python
-import nibabel as nib 
-import numpy as np 
-import scipy.spatial 
+import nibabel as nib
```

Comment:
Should get rid of this file too

Reply:
This is already gone in the `main` branch. Actually `split-merge` needs a rebase from main, or would you rather I just make this PR to `main` and delete the other branch?

Reply:
Let's just delete the other branch, not a lot of value in it anymore.